projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -639,18 +649,14 @@ export class IgxForOfDirective<T, U extends T[] = T[]> extends IgxForOfToken<T,U
         if (nextScroll < 0) {
             return;
         }
-        if (this.igxForScrollOrientation === 'horizontal') {
-            this.scrollPosition = this.isRTL ? -nextScroll : nextScroll;
```

Comment:
The standard scroll should still work since there's a test for RTL. But not sure if there are tests for the api methods.  Might be better to just invert it in the scrollPosition setter IMO, so that we don't have to repeat this `isRTL` check in every api method.